src/secp256k1.c

```diff
@@ -120,7 +120,7 @@ static int secp256k1_pubkey_load(const secp256k1_context* ctx, secp256k1_ge* ge,
          * representation inside secp256k1_pubkey, as conversion is very fast.
          * Note that secp256k1_pubkey_save must use the same representation. */
         secp256k1_ge_storage s;
-        memcpy(&s, &pubkey->data[0], 64);
+        memcpy(&s, &pubkey->data[0], sizeof(secp256k1_ge_storage));
```

Comment:
I actually do have a preference ordering of:

memcpy(&s, ..., <numeric constant>)    <   memcpy(&s, ..., sizeof(type))   <  memcpy(&s, ..., sizeof(s))

Will update to sizeof(s), was an oversight on my part.